sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/SessionCatalog.scala

```diff
@@ -49,6 +49,8 @@ class SessionCatalog(
     hadoopConf: Configuration) extends Logging {
   import CatalogTypes.TablePartitionSpec
 
+  val DEFAULT_DATABASE = "default"
```

Comment:
Oh, I remember why I didn't do that. In fact, we don't have `object SessionCatalog`, yet.
So, I'm not sure I can introduce new object for just one constant variable. Is it enough valuable, @srowen  and @rxin .